.github/workflows/standard_library_tests_and_examples.yml

```diff
@@ -25,6 +25,7 @@ permissions:
 jobs:
   test-examples:
     strategy:
+      fail-fast: false
```

Comment:
I'm assuming this was left here accidentally, right?

Reply:
I had another PR merged into this one to make the debug easier. It got merged into nightly. So this diff should go away now.